src/vec.rs

```diff
@@ -1180,18 +1952,41 @@ where
     A: PartialEq<B>,
 {
     fn eq(&self, other: &Vec<A, N>) -> bool {
-        <[A]>::eq(other, &self[..])
+        other.as_view().eq(self)
+    }
+}
+
+// &[B; M] == VecView<A>
+// Equality does not require equal capacity
+impl<A, B, const M: usize> PartialEq<VecView<A>> for &[B; M]
+where
+    A: PartialEq<B>,
+{
+    fn eq(&self, other: &VecView<A>) -> bool {
+        <[A]>::eq(other, *self)
     }
 }
 
 // Implements Eq if underlying data is Eq
 impl<T, const N: usize> Eq for Vec<T, N> where T: Eq {}
+// Not sure about this one? Should two distinct capacities really be Eq?
+// Anyways it derefs to &[u8] which implements Eq so I suppose it should
```

Comment:
You can remove this comment. We already have `Eq` for `Vec` itself, so it makes sense to also have it for `VecView`. We should probably also have `Eq` for `Vec`s with different capacities.

Reply:
Ok